Apromore-Core-Components/Apromore-Portal/pom.xml

```diff
@@ -24,6 +24,7 @@
 
     <properties>
         <junit.jupiter.version>5.7.0</junit.jupiter.version>
+        <keycloak.version>12.0.4</keycloak.version>
```

Comment:
This property is inherited already from ApromoreCore/pom.xml, so I think it's redundant here.